templates/rhpam70-kieserver-mysql.yaml

```diff
@@ -0,0 +1,535 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    description: Application template for a managed KIE server with a MySQL database, for Red Hat Process Automation Manager 7.0
+    iconClass: icon-jboss
+    tags: rhpam,jboss,xpaas
+    version: 1.4.0
+    openshift.io/display-name: Red Hat Process Automation Manager 7.0 managed KIE server with a MySQL database
+  name: rhpam70-kieserver-mysql
+labels:
+  template: rhpam70-kieserver-mysql
+  xpaas: 1.4.0
+message: A new environment has been set up for Red Hat Process Automation Manager 7. The username/password for accessing the KIE server is ${KIE_SERVER_USER}/${KIE_SERVER_PWD}.
+parameters:
+- displayName: Application Name
+  description: The name for the application.
+  name: APPLICATION_NAME
+  value: myapp
+  required: true
+- displayName: Maven repository URL
+  description: Fully qualified URL to a Maven repository or service.
+  name: MAVEN_REPO_URL
+  example: http://nexus.nexus-project.svc.cluster.local:8081/nexus/content/groups/public/
+  required: true
```

Comment:
It should not be required, customer may not have a maven repo available or the idea of this template is not integrate with business central?

Reply:
No, this would not connect to an authoring environment. I think it needs to be required. Not sure how else it can run anything, without a maven repo.

Reply:
What if it's an s2i case with no external dependencies other than maven central?

Reply:
We have a different template for s2i. This one is not set up to support that, have Git / buildconfig / etc.

Reply:
Gotcha. :+1: